rviz_marker_tools/package.xml

```diff
@@ -9,9 +9,26 @@
 
 	<buildtool_depend>catkin</buildtool_depend>
 
-	<depend>visualization_msgs</depend>
-	<depend>geometry_msgs</depend>
-	<depend>roscpp</depend>
-	<depend>tf2_eigen</depend>
-	<depend>liburdfdom-dev</depend>
+	<build_depend>eigen</build_depend>
+	<build_export_depend>eigen</build_export_depend>
+
+	<build_depend>geometry_msgs</build_depend>
+	<build_export_depend>geometry_msgs</build_export_depend>
+	<exec_depend>geometry_msgs</exec_depend>
```

Comment:
Why do you prefer this verbose form over `<depend>`?

Reply:
I just preferred to stay with one style and some of the others don't use all three lines.
I collapsed the relevant lines now.